dedicated-examples/mock-camera-microphone-access/README.md

```diff
@@ -1 +1,29 @@
-# mock camera/microphone access
+# Mock Camera/Microphone Access
+
+This example shows how to mock data for microphone and camera. This example runs in **Chrome**.
```

Comment:
This example runs in Chrome and shows how to use mock data for the microphone and camera.